web/client/components/map/openlayers/VectorStyle.js

```diff
@@ -333,7 +333,7 @@ const getValidStyle = (geomType, options = { style: defaultStyles}, isDrawing, t
                 image: options.style.useSelectedStyle || options.style.highlight ? new ol.style.Circle({
                     radius: 3,
                     fill: new ol.style.Fill(tempStyle.fill ? tempStyle.fill : {
-                        color: colorToRgbaStr(options.style && tempStyle.color || "#0000FF", tempStyle.opacity || 0.2)
+                        color: blue
```

Comment:
always blue? I would prefer green

Reply:
Blue is chosen to be color of the stroke when the feature is selected. 
this style is only applied when options.style.useSelectedStyle is true, and it must be blue not the feature's color.